lib/services/ngsiParser.js

```diff
@@ -211,10 +211,13 @@ function getErrorField(body) {
     var errorField = body.errorCode ||
         body.orionError;
 
-    if (body && body.contextResponses &&
-        body.contextResponses[0] && body.contextResponses[0] &&
-        body.contextResponses[0].statusCode && body.contextResponses[0].statusCode.code !== '200') {
-        errorField = body.contextResponses[0].statusCode;
+    if (body && body.contextResponses) {
+        for (var i in body.contextResponses) {
+            if (body.contextResponses[i] && body.contextResponses[i] &&
```

Comment:
Fixing. I'm even going to remove both, as that check is implicit in the foreach.